apps/mark-scan/backend/src/custom-paper-handler/state_machine.ts

```diff
@@ -324,15 +369,86 @@ export function buildMachine(
           id: 'interpretScannedBallot',
           src: loadMetadataAndInterpretBallot,
           onDone: {
-            target: 'presenting_ballot',
+            target: 'transition_interpretation',
             actions: assign({
-              interpretation: (_, event) => {
-                return event.data;
-              },
+              interpretation: (_, event) => event.data,
             }),
           },
         },
       },
+      // Intermediate state to conditionally transition based on ballot interpretation
+      transition_interpretation: {
+        always: [
+          {
+            target: 'presenting_ballot',
+            cond: (context) =>
+              context.interpretation
+                ? context.interpretation[0].interpretation.type ===
+                  'InterpretedBmdPage'
+                : false,
+          },
+          {
+            target: 'blank_page_interpretation',
+            cond: (context) =>
+              context.interpretation
+                ? context.interpretation[0].interpretation.type === 'BlankPage'
+                : false,
+          },
+        ],
+      },
+      blank_page_interpretation: {
+        invoke: pollPaperStatus(),
+        entry: async (context) => {
+          await context.driver.presentPaper();
+        },
+        initial: 'presenting_paper',
+        // These nested states differ slightly from the top-level paper load states so we can't reuse the latter.
+        states: {
+          presenting_paper: {
+            // Heavier paper can fail to eject completely and trigger a jam state even though the paper isn't physically jammed.
+            // To work around this, we avoid ejecting to front. Instead, we present the paper so it's held by the device in a
+            // stable non-jam state. We instruct the poll worker to remove the paper directly from the present state.
+            entry: async (context) => await context.driver.presentPaper(),
+            on: { NO_PAPER_ANYWHERE: 'accepting_paper' },
+          },
+          accepting_paper: {
+            on: {
+              PAPER_READY_TO_LOAD: 'load_paper',
+            },
+          },
+          load_paper: {
+            entry: async (context) => {
+              await context.driver.loadPaper();
+              await context.driver.parkPaper();
+            },
+            on: {
+              PAPER_PARKED: 'done',
+              NO_PAPER_ANYWHERE: 'accepting_paper',
+            },
+          },
+          done: {
+            type: 'final',
+            entry: () => {
+              assign({
+                interpretation: undefined,
+                scannedImagePaths: undefined,
+              });
+            },
+          },
+        },
+        onDone: { target: 'paper_reloaded' },
+      },
+      // paper_reloaded is a distinct state from paper_parked only
+      // because the frontend handles them differently
```

Comment:
`paper_reloaded` is only used by the `blank_page_interpretation` state and could be a child state of `blank_page_interpretation`, but there are 2 frontend components.
1. `BlankPageInterpretationPage` for all child states currently in `blank_page_interpretation`
2. `PaperReloadedPage` for the `paper_reloaded` state

So our conditions on `stateMachineState` are very concise; see https://github.com/votingworks/vxsuite/pull/3981/files#diff-dc3b4ba8825793529470eb8c571a54a06cff23c83696464aebf5bea255070ec5R702

If we make `paper_reloaded` a child of `blank_page_interpretation` then we'd have to check states like:

```
if (['blank_page_interpretation.presenting_paper', 'blank_page_interpretation.accepting_paper', 'blank_page_interpretation.load_paper'].includes(stateMachineState)) {
  return <BlankPageInterpretationPage ... />
}
if (stateMachineState === 'blank_page_interpreation.paper_reloaded') {
  return <PaperReloadedPage />
}
```